frontend/src/index.css

```diff
@@ -1,3 +1,19 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer components {
+  .scale-hover:hover {
+    transform: scale(1.12);
+  }
+
+  @media (min-width: 640px) {
+    .sm\:scale-hover:hover {
+      transform: scale(1.12);
+    }
+  }
+
```

Comment:
Why are we doing explicit declaration ? , `sm:scale-hover` should ideally work without the above @media code in my opinion, And if not, is it the right way? it is almost like your classname is `sm:scale-hover` you have removed significance of `sm` in your class `sm:scale-hover:hover`.